### DISCUSSION TITLE
PlayerMoveEvent alternative

### DISCUSSION THREAD

**Discussion by zax71** [Minestom/Minestom#1701] [Help] [answered]
I’ve made a “death” barrier that teleports players to another position when passing past that point using the player move event but that is known to be resource intensive, so, is there a better way to check if the player is in a specific position?

**Comment by emortaldev** [comment, marked as answer]
It definitely isn't resource intensive, your only other alternative is PlayerTickEvent which is more reliable and I suppose ***very slightly*** slower

but it's really nothing to worry about

EDIT: Technically, you can make a repeating task that repeats every x seconds, however it is still *definitely* not worth worrying about speed when you're doing a simple Y level check

**Reply by zax71** [reply]
Thanks for that, I guess my old Bukkit-API knowledge is creeping in
